### DISCUSSION TITLE
Does Saelor work with renting products?

### DISCUSSION THREAD

**Discussion by Infiniti20** [saleor/saleor#8912] [Q&A]
I'm assuming not. I was working on a personal project, and a friend recommended Saelor to me. I don't see anything in the docs about renting.

**Comment by karolkielecki** [comment]
@Infiniti20 Hi! Could you please share more details and a use case?

**Reply by Infiniti20** [reply]
I have a open marketplace website where anybody can add products. I have code for determining availability already, but I'm worried about edge cases and the problems they could create. A friend recommended Saelor to me, since I was currently managing everything myself. I can't seem to find anything about renting on Saelor, so it seems I'll have to look elsewhere.

**Reply by karolkielecki** [reply]
Thanks for more details. Saleor doesn't support such functionality -- marketplace/multivendor. We have considered marketplace functionality in the past, but we do not plan to implement it. However, we do have an integration with Jetti.io - https://jetti.io/integrations/store/saleor. You might check it out as a possible fit.

**Comment by c-nv-s** [comment]
I'm researching the feasibility of using saleor for renting items, just curious if there is any future plans or existing guidance on how to include time/duration availability to items. In my mind I have a possible workarounds either by using Apollo Federation or somehow hacking the shipping events webhook, but it would be good to know if there will ever be an officially supported feature for this? it seems a few people have asked now.
( #8975 )

**Reply by timuric** [reply]
Hi @c-nv-s this is quite specific use case that you would need to orchestrate yourself. I suggest to build independent service for that or an app if you also need admin experience
